IceCubesApp/App/Tabs/Settings/DisplaySettingsView.swift

```diff
@@ -147,22 +149,36 @@ struct DisplaySettingsView: View {
       .navigationDestination(isPresented: $isFontSelectorPresented, destination: { FontPicker() })
 
       VStack {
-        Slider(value: $theme.fontSizeScale, in: 0.5 ... 1.5, step: 0.1)
-        Text("settings.display.font.scaling-\(String(format: "%.1f", theme.fontSizeScale))")
+        Slider(value: $fontScale, in: 0.5 ... 1.5, step: 0.1) { editing in
```

Comment:
I think this should be done like I've solved lagging for colors. 
Look at `DisplaySettingsLocalColors`
Probably rename it to DisplaySettingsLocalValues.
And connect the slider values the same way. This way we debounce them and the slider is not laggy.

Reply:
Implemented it

Reply:
Perfect thanks!